rpc/js/package.json

```diff
@@ -8,7 +8,8 @@
   },
   "devDependencies": {
     "@types/google-protobuf": "^3.7.4",
-    "prettier": "^2.8.8",
+    "@viamrobotics/prettier-config": "^0.3.4",
+    "prettier": "^3.0.3",
```

Comment:
There's a `format` script below that has `{ts,js}`. We'll want to add `cjs` to that list since we've added `.prettierrc.cjs`